implementations/rust/ockam/ockam_command/tests/bats/unit.bats

```diff
@@ -358,6 +358,38 @@ teardown() {
   assert_output --partial "NotFound"
 }
 
+@test "portals - delete a tcp inlet" {
+  outlet_port=5000
+  inlet_port=6000
```

Comment:
Make sure you don't use the same port in different tests, as they could be run in parallel.

Reply:
Switched around the portals ports I'd been using